### PR TITLE
Split between OrientationSensor and  AbsoluteOrientationSensor

### PR DESCRIPTION
**Pull request by pozdnyakov**
As follows from https://github.com/w3c/orientation-sensor/issues/15

***
[Preview](https://s3.amazonaws.com/pr-preview/pozdnyakov/orientation-sensor/split_OrientationSensor_and_AbsoluteOrientationSensor.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/orientation-sensor/bd7beee...pozdnyakov:5d602e2.html)

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -105,31 +105,31 @@ Introduction {#intro}
...
+in relation to a three dimensional Cartesian coordinate system.
 
-The base object describes the orientation in relation to <a>Earth's reference
-coordinate system</a>, thus in a stationary three dimensional Cartesian
-coordinate system.
+The AbsoluteOrientationSensor API inherits from the OrientationSensor API and
+describes the device's physical orientation in relation to <a>Earth's reference coordinate system</a>.
 
-Inheriting objects can describe the orientation in relation to other stationary
+Other inherited objects can describe the orientation in relation to other stationary
```
Comment (alexshalamov) [inline index.bs]: > Other inherited objects

Other subclasses or inherited interfaces?

File: index.bs [hunk 2]
```diff
@@ -11,8 +11,8 @@ Editor: Kenneth Rohde Christiansen 57705, Intel Corporation, http://intel.com/
 Editor: Anssi Kostiainen 41974, Intel Corporation, http://intel.com/
 Group: dap
 Abstract:
-  This specification defines concrete sensor interfaces to monitor the device's physical
-  orientation in relation to <a>Earth's reference coordinate system</a>.
+  This specification defines a generic interface and inherited concrete sensor interfaces to monitor the device's
```
Comment (alexshalamov) [inline index.bs]: >  generic interface and inherited

base orientation sensor interface and concrete sensor subclasses?

**Comment by kenchris** [comment]
Lgtm

File: index.bs [hunk 3]
```diff
@@ -105,31 +105,31 @@ Introduction {#intro}
...
 
 The OrientationSensor API extends the Generic Sensor API [[GENERIC-SENSOR]]
-to provide information that describes the device's physical orientation.
+to provide generic information describing the device's physical orientation
+in relation to a three dimensional Cartesian coordinate system.
 
-The base object describes the orientation in relation to <a>Earth's reference
-coordinate system</a>, thus in a stationary three dimensional Cartesian
-coordinate system.
+The AbsoluteOrientationSensor API inherits from the OrientationSensor API and
```
Comment (anssiko) [inline index.bs]: More clear and consistent with links: {{AbsoluteOrientationSensor}} {{OrientationSensor}}

File: index.bs [hunk 4]
```diff
@@ -105,31 +105,31 @@ Introduction {#intro}
...
+describes the device's physical orientation in relation to <a>Earth's reference coordinate system</a>.
 
-Inheriting objects can describe the orientation in relation to other stationary
+Other subclasses can describe the orientation in relation to other stationary
```
Comment (anssiko) [inline index.bs]: s/can describe/describe/

File: index.bs [hunk 5]
```diff
@@ -105,31 +105,31 @@ Introduction {#intro}
...
 directions, such as true north, or even non stationary directions, like in
```
Comment (anssiko) [inline index.bs]: s/or even/or/

Better be explicit. Reword as needed.

File: index.bs [hunk 6]
```diff
@@ -105,31 +105,31 @@ Introduction {#intro}
...
 relation to a devices own z-position, drifting towards its latest most stable
 z-position.
 
-The data provided by the OrientationSensor API are similar to data from [=deviceorientation Event=], but the OrientationSensor
-API has some significant differences:
+The data provided by the OrientationSensor-based APIs are similar to data from
```
Comment (anssiko) [inline index.bs]: {{OrientationSensor}}

File: index.bs [hunk 7]
```diff
@@ -105,31 +105,31 @@ Introduction {#intro}
...
+[=deviceorientation Event=], but the OrientationSensor API has some significant differences:
 1. The OrientationSensor API represents orientation data in WebGL-compatible formats (quaternion, rotation matrix).
 1. The OrientationSensor API satisfies stricter latency requirements.
-1. Unlike [=deviceorientation Event=] the OrientationSensor API explicitly defines which [=low-level=] motion sensors are used
-   to obtain the orientation data, thus obviating possible interoperability issues.
-1. An {{OrientationSensor}} instance is configurable via {{SensorOptions}} constructor parameter.
+1. Unlike [=deviceorientation Event=] the OrientationSensor-based APIs explicitly define which [=low-level=]
```
Comment (anssiko) [inline index.bs]: {{OrientationSensor}}

**Comment by pozdnyakov** [comment]
@anssiko PTAL

**Comment by anssiko** [comment]
LGTM

**Pull request merged**